scripts/reduce_CG1D.py

```diff
@@ -90,8 +91,10 @@ def main(inputfile: Union[str, Path], outputdir: Union[str, Path]) -> int:
 
     # step 2: extract info from inputfile
     update_dict = extract_info_from_path(str(inputfile))
+    assert update_dict["instrument"] == "CG1D", "Instrument is not CG1D"
 
     # step 3: update the dict and save dict to disk
+    substitute_template(config_dict, update_dict)
```

Comment:
Should the return value here be assigned to anything?

Reply:
yes, it should be:
```python
config_dict = substitute_template(config_dict, update_dict)
```